openshift/4.0/templates/Logging/oclogbc.yaml

```diff
@@ -0,0 +1,135 @@
+# This is a build template for sidecar logging
+# It will build a new image from the specified source control repo
+# that will run your application.
+kind: Template
+apiVersion: v1
+metadata:
+  name: pims-app-log-build
+  annotations:
+    openshift.io/display-name: pims log sidecar
+    description: openshift sidecar - will collect oc logs and submit a zip to an endpoint.
+    tags: pims,log
+parameters:
+  - name: APP_NAME
+    displayName: "App Name"
+    description: "The name of the application (grouped)."
+    required: true
+    value: "pims-logging"
+  - name: ROLE_NAME
+    displayName: "Component Role Name"
+    description: "The name of the application component"
+    required: true
+    value: "pimslog"
+  - name: PROJECT_NAMESPACE
+    displayName: "OpenShift Project Namespace"
+    description: "The namespace of the OpenShift project containing the application."
+    required: true
+    value: "3cd915"
+  - name: ENV_NAME
+    displayName: "Environment name"
+    description: "The name for this environment [dev, test, prod, tools]"
+    required: true
+    value: "tools"
+
+  - name: GIT_URL
+    displayName: "Git Repository URL"
+    description: "The URL of the repository with your application source code."
+    required: true
+    value: "https://github.com/sookeke/PSP.git"
```

Comment:
Default git_url value should point to **bcgov** repo not a user fork

Reply:
true will change to bcgov